File: Leetcode/Greedy/1899. Merge Triplets to Form Target Triplet.py
class Solution:
    def mergeTriplets(self, triplets: List[List[int]], target: List[int]) -> bool:
        x, y, z = 0, 0, 0
        for a, b, c in triplets:
            if a <= target[0] and b <= target[1] and c <= target[2]:
                x, y, z = max(x, a), max(y, b), max(z, c)
        return [x, y, z] == target

File: Leetcode/Greedy/1899. Merge Triplets to Form Target Triplet.cpp
class Solution {
public:
    bool mergeTriplets(vector<vector<int>>& triplets, vector<int>& target) {
        vector <int> Answer = {0, 0, 0};
        for (vector <int> &triplet: triplets) {
            if (triplet[0] <= target[0] && triplet[1] <= target[1] && triplet[2] <= target[2]){
                Answer[0] = max(Answer[0], triplet[0]);
                Answer[1] = max(Answer[1], triplet[1]);
                Answer[2] = max(Answer[2], triplet[2]);
            }
        }
        return target == Answer;
    }
};